model/user/db.go

```diff
@@ -488,18 +488,31 @@ func ClearLoginCache(user gimlet.User) error {
 
 // ClearUser clears one user's settings, roles, and login cache.
```

Comment:
we should update this to explain why we only clear some settings.

Reply:
Let me know if I'm too wordy... Thought it better so no one has to investigate whyy this function does what it does.